### ISSUE TITLE
Add individual pages for each changelog entry

### ISSUE THREAD

**Issue by gtsiolis** [gitpod-io/website#681]
### Problem to solve

Naturally, our [changelog](https://www.gitpod.io/changelog) will grow and we'll have to add pagination at some point. It would be nice to be able to link directly to a changelong entry in a standalone page instead of having to use an anchor link to the heading, like [the last release](https://www.gitpod.io/changelog#faster-incremental-prebuilds), which will not be possible once we add pagination.

### Proposal

Having a permalink to each entry with a structure like the following could suffice:

- `/changelog/2021-05-31-faster-incremental-prebuilds`

TBD (To be discussed)

Cc @mikenikles @JohannesLandgraf

**Comment by mikenikles** [comment]
I agree, this is even more critical as we start to ship releases more frequently 🚀. I propose we keep `/changelog` the way it is and introduce pagination when page load time starts to become an issue, but make the headings and/or dates links to individual changelog pages with the URL format you suggested.

This gives us the best of both worlds: A single page to scroll through and see what's new and individual pages we can use to share if we want to talk about a specific release.
